docs/notebooks/2_getting_started.ipynb

```diff
@@ -38,10 +38,10 @@
     "\n",
```

Comment:
<p>I don't like that we have to do this, but if we really can't have bigger samples I guess it's ok..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>The way that I see it, we have 3 options:</p><ol><li>What I've done here, which is to call <code>extract_dataset</code>, then highjack the results with some fake data.</li><li>Completely separate the <code>extract_dataset</code> demonstration into its own section or notebook, and use <code>xclim.testing.open_dataset()</code> from <code>regrid</code> onwards.</li><li>Use a PAVICS catalog, but then we'd have to download data every time (which is probably not great?). I'm also not 100% convinced that the current form of <code>extract_dataset</code> would work, but we could test it out.</li></ol>

Reply:
@aulemahal Thoughts?

Reply:
For 2, If you create all the fake data in one section, you can add it to the project catalogue and still open it from the catalog in the next sections, no?

Reply:
<p>Hum, I agree with Juliette here : couldn't you create the fake data first and then use that in the notebook ?</p><p>We can hide cells from the RTD documentation, so you could create netCDFs on the fly using <code>datablock</code> in a hidden cell on top and then use those files normally.</p>

Reply:
<p>https://nbsphinx.readthedocs.io/en/0.9.3/hidden-cells.html</p><p> </p>